include/ola/testing/TestUtils.h

```diff
@@ -34,12 +34,19 @@ namespace ola {
 namespace testing {
 
 // Assert that two data blocks are the same.
+// Use OLA_ASSERT_DATA_EQUALS below.
```

Comment:
Is that something we should try and fix though and switch to the macro? Presumably people have to manually set __LINE__ etc?